java/ql/lib/semmle/code/java/security/CleartextStoragePropertiesQuery.qll

```diff
@@ -5,14 +5,6 @@ import semmle.code.java.dataflow.DataFlow
 import semmle.code.java.frameworks.Properties
 import semmle.code.java.security.CleartextStorageQuery
 
-private class PropertiesCleartextStorageSink extends CleartextStorageSink {
```

Comment:
The class `CleartextStorageSink` is abstract, so this construction actually "extends" that class, i.e. it adds to it. So this piece of code is very much in use.